mm2src/coins/Cargo.toml

```diff
@@ -183,6 +183,10 @@ zcash_proofs = { workspace = true, features = ["local-prover", "multicore"] }
 [target.'cfg(windows)'.dependencies]
 winapi.workspace = true
 
+# iOS-specific dependencies to fix certificate validation issues
+[target.'cfg(target_os = "ios")'.dependencies]
```

Comment:
since we re adding this as a iOS specific override this means both are active since rustls-tls-native-roots is enabled by the gen dependency. so if we want to differentiate platform wise we d need "per target" sections for diff. between native and webpki. 

alternatively (how i tested in a local fix): we just update it to webpki globally cc @onur-ozkan whats preferred / more solid from?

Reply:
If we *must* use webpki for IOS, I suggest doing it for IOS only and keep native roots for others (as we have no issues with it, at least so far?). Relying on OS roots are better than relying on the roots that are bundled with the dependency which will not be updated for a quite long time (we don't update our deps very often).

Reply:
thanks - will update this PR accordingly to handle it platform-specific shortly and keep the native everywhere else as there weren't any issues with it (yet)